tests/providers/amazon/aws/hooks/test_s3.py

```diff
@@ -467,7 +466,6 @@ async def test_s3_key_hook_get_head_object_with_error_async(self, mock_client):
 
     @async_mock.patch("airflow.providers.amazon.aws.triggers.s3.S3Hook.async_conn")
     @pytest.mark.asyncio
-    @unittest.expectedFailure
```

Comment:
This one was added into the initial commit: https://github.com/apache/airflow/pull/31018

So there is two option:
1. It was add by a mistake
2. The test is broken from the initial commit, which are not good.

There is no error when I tried to run it locally, however better to check it into the CI

Reply:
I guess maybe it was added because it failed initially if aiobotocore not installed

Reply:
likely